page/ts/index.ts

```diff
@@ -69,21 +79,17 @@ $(document).ready(function() {
   $(".person-block a").click(function(e) {
     e.preventDefault();
   });
+
   $(".person-block").click(function() {
-    const name: string = $(this).find("h4.name a").text();
-    const image: string = $(this).find("img").attr("src");
-    const title: string = $(this).find("h4.position").text();
-    const bio: string = $(this).find("p.bio").text();
-    const preTitle: string = $(this).find("span.pre-title").text();
-    const domain: string = $(this).find("a.domain").text();
-    const email: string = $(this).find("p.link").text();
-
-    vex.dialog.open(getPersonModal(name, image, preTitle, title, bio, domain, email));
-  });
+    const name: string = $(this).find("h4.name a").text().trim();
```

Comment:
i am pretty sure that `:string` here is not needed. It's inferred by the compiler. Could you try removing it?